sql/moz-fx-data-shared-prod/mozilla_org_derived/ga_sessions_v2/script.sql

```diff
@@ -0,0 +1,415 @@
+MERGE INTO
+  `moz-fx-data-shared-prod.mozilla_org_derived.ga_sessions_v2` T
```

Comment:
An explanation of why this is using `MERGE INTO` would be very helpful!

Reply:
This actually was already taken out in a subsequent PR